zend_abstract_interface/methods/tests/php5/methods.cc

```diff
@@ -737,6 +737,8 @@ TEST_CASE("call static method: non-static method (userland)", "[zai_methods]") {
     REQUIRE(retval != NULL);
     REQUIRE(Z_TYPE_P(retval) == IS_BOOL);
 
+    zval_ptr_dtor(&retval);
```

Comment:
I assume this was found via the added memory leak detection?

Reply:
It was. :)